packages/components/form/src/form-item.vue

```diff
@@ -281,29 +254,47 @@ export default defineComponent({
         .map((rule) => ({ ...rule }))
     }
 
-    const evaluateValidationEnabled = () => {
-      isValidationEnabled.value = !!getRules()?.length
-    }
+    const evaluateValidationEnabled: FormItemContext['evaluateValidationEnabled'] =
+      () => {
+        isValidationEnabled.value = !!getRules()?.length
+      }
 
-    const updateComputedLabelWidth = (width: string | number) => {
-      computedLabelWidth.value = width ? `${width}px` : ''
-    }
+    const updateComputedLabelWidth: FormItemContext['updateComputedLabelWidth'] =
+      (width) => {
+        computedLabelWidth.value = width ? `${width}px` : ''
```

Comment:
Let's rewrite the update statement here.

Reply:
I didn't get your point. How to rewrite?

Reply:
I didn't get your point. How to rewrite?